CMakeLists.txt

```diff
@@ -135,6 +135,14 @@ if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
         FORCE)
 endif()
 
+# If set, make MBEDTLS_CONFIG_FILE and MBEDTLS_USER_CONFIG_FILE into PATHs
+if(DEFINED MBEDTLS_CONFIG_FILE)
```

Comment:
Do we need these `if(DEFINED ...)` blocks due to an older version of CMake?

Reply:
We do not, I had just forgotten to update the backport. This works the same way according to the docs I have for CMake 2.8.12, which is the oldest supported for this branch.